Spec/Auth.swift

```diff
@@ -1679,6 +1675,7 @@ class Auth : QuickSpec {
             // RSA10a
             it("should create a token if needed and use it") {
```

Comment:
Also, https://github.com/ably/ably-ios/pull/527/files#diff-b5d84fe9bce26902a7481c2ffec84fffR1673 should be `authorize`